advanced/tts/tts_main.py

```diff
@@ -226,12 +226,16 @@ def tts(
         if tts_model.is_multi_lingual and language is None:
             raise HTTPException(
                 status_code=400,
-                detail="Model is multi-lingual, you should pass in language.",
+                detail="Model is multi-lingual, you should pass in language. \
```

Comment:
nit: we don't use backslash to do multiple line strings. Use

```
pip install -U black
black --preview .
```

to auto-format?